chukwuemekachm/M-Tracker-React-Client
# #160876412 Refactors the dashboard UI to use bootstrap

#### What does this PR do?
- This PR refactors the dashboard UI to use bootstrap
#### Description of Task to be completed?
- Refactor SideNav.jsx
- Refactor UserRequest.jsx
- Refactor Dashboard.jsx
- Refactor tests
- Update snapshots
- Remove CreateRequest.jsx
- Remove RequestDetails.jsx
- Remove RequestDetailsPage.jsx
#### How should this be manually tested?
- Clone the repo
- Startup command line/Terminal
- Navigate to the project's directory
- Run `npm install`
- Run `npm start` or `npm run dev`
- Your default browser will be open if you used `npm run dev`
- Login or signup
- Observe the new state of the dashboard UI
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#160876412](https://www.pivotaltracker.com/story/show/160876412)
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-10-01 at 11 20 25 am" src="https://user-images.githubusercontent.com/33798252/46283534-8bf48d80-c56c-11e8-9f0b-c5ed74f72682.png">


<img width="1440" alt="screen shot 2018-10-01 at 11 21 12 am" src="https://user-images.githubusercontent.com/33798252/46283535-8bf48d80-c56c-11e8-9b18-ab30a86d901a.png">

#### Questions:
- None
